### DISCUSSION TITLE
Stream feeds show locked.

### DISCUSSION THREAD

**Discussion by tuxydoh** [mrlt8/docker-wyze-bridge#1316] [Q&A]
I got everything running (thanks for help) and now when it loads in docker it just has a still image. Enabling streaming does not seem to function and none of the stream feed urls are functioning. Been searching for a while and cannot seem to find an answer.
![2024-08-08_08-58-25](https://github.com/user-attachments/assets/559e00ca-f601-47f9-b201-1de1768d0ed3)

**Comment by mrlt8** [comment]
Can you try running the container with [authentication disabled](https://github.com/mrlt8/docker-wyze-bridge/wiki/Authentication) `- WB_AUTH=False`.

**Reply by tuxydoh** [reply]
Sorry I am a super noob not sure how to do that

**Reply by mrlt8** [reply]
If you're running the bridge in Home Assistant it should be under the add-on's config. If you're using `docker-compose.yml`, you'll need to add or update the line `WB_AUTH=False` under `environment`:
https://github.com/mrlt8/docker-wyze-bridge/blob/370a13839f519128791023c10114b30319983ed7/docker-compose.sample.yml#L24

**Reply by Basdub** [reply]
hi, i had the same issue, removing the webui auth as help me gain access to the live feed using the hls option. thanks for this.

Its choppy but decent.

I was wondering if i should be able to have a live feed in the previous screen where all the cameras are listed.

When i open the stream menu and click play it shows an error message at the top right of screen.
![Screenshot 2024-09-09 100906](https://github.com/user-attachments/assets/0510a4b9-e35f-46ba-b40b-68b042c17d17)

**Reply by mrlt8** [reply]
You probably want to select one of the video options at the bottom of the page:
<img width="355" alt="video" src="https://github.com/user-attachments/assets/e198c8e1-38c7-4d8c-84bc-db21a5ce94c3">
